knora-ontologies/knora-base.ttl

```diff
@@ -2703,11 +2920,99 @@ rdfs:label rdfs:subPropertyOf :objectCannotBeMarkedAsDeleted .
                                                "Representation (Text)"@en ,
                                                "Répresentation (Texte)"@fr ,
                                                "Rappresentazione (testo)"@it ;
-                    
+
                     rdfs:comment "A resource containing text files"@en .
 
 
 
+###  http://www.knora.org/ontology/knora-base#Person
+
+:Person rdf:type owl:Class ;
+
+        rdfs:subClassOf :Resource, foaf:Person,
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasFamilyName ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasGivenName ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasIAFIdentifier ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasBirthDate ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasDeathDate ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ];
+
+        rdfs:label "Person"@de ,
+                   "Person"@en ,
+                   "Personne"@fr ,
+                   "Persona"@it ;
+
+        rdfs:comment "A resource representing a person"@en .
+
+
+###  http://www.knora.org/ontology/knora-base#Publication
+
+:Publication rdf:type owl:Class ;
```

Comment:
We put that in knora-base because we want to reuse existing vocabularies (see <http://link.springer.com/chapter/10.1007/978-3-319-04777-5_5>) and further refine them in BEOL.

The difficulty of doing that lies in Knora's subject and object class constraint (see our discussion about the DC). But I also think that once you want to define things that are used over more than one project, you have to think about certain commonalities which makes you less flexible in your own project.

But yes, nothing speaks against having several general ontologies.